test/integration/mapping_validation_test.rb

```diff
@@ -20,6 +20,35 @@ class MappingValidationTest < ActiveSupport::TestCase
       #{invalid_categories}"
   end
 
+  test "every Shopify category has corresponding channel mappings" do
+    mappings_json_data = CLI.new.parse_json("dist/en/integrations/all_mappings.json")
```

Comment:
Should we move this to a `before_all` to save parsing a very large file multiple times?

Reply:
I addressed all the other comments.  `before_all` seems not applicable to this test suite because it aims to speed up testing for specs that share expensive database setup code according to the [minitest-hooks](https://github.com/jeremyevans/minitest-hooks) but this test suite does not involve db setup.  Instead, I defined `def setup` to include steps shared by different test cases.

Reply:
I think parsing a massive YAML file _is slow_. Happy to be wrong on it speeding things up, but we use it for _exactly this_ in our other integration test